i18n/ja/docusaurus-plugin-content-docs/current/cache/develop/sdks/dart/cheat-sheet.mdx

```diff
@@ -0,0 +1,74 @@
+---
```

Comment:
i think we are not supposed to commit stuff into the i18n directory, I think the translator bot is supposed to do that. i'd probably leave this stuff out for now and we can ask erika to take a peek? there is already a bit of backlog for getting the translations up to date.

but if you know things about this directory that i don't, lmk!

Reply:
Ok, I'm happy to remove these. I wasn't sure how this process was supposed to work.